staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -290,6 +290,15 @@ func (f *StorageFactoryRestOptionsFactory) GetRESTOptions(resource schema.GroupR
 	return ret, nil
 }
 
+// DefaultWatchCacheSizes defines default resources for which watchcache
+// should be disabled.
+func DefaultWatchCacheSizes() map[schema.GroupResource]int {
+	return map[schema.GroupResource]int{
+		{Resource: "events"}:                         0,
+		{Group: "events.k8s.io", Resource: "events"}: 0,
```

Comment:
this probably belongs in kube-apiserver, not in k8s.io/apiserver

Reply:
done